certified-connectors/bitskout/apiDefinition.swagger.json

```diff
@@ -163,7 +163,737 @@
           }
         ]
       }
-    }
+    },
+    "/actions/invoices": {
```

Comment:
@saferoombiz all files should be properly formatted. please fix the indentation for the swagger file.